RunningOrder/Story/StoryList.swift

```diff
@@ -13,24 +13,22 @@ extension Story: Identifiable {
 }
 
 struct StoryList: View {
```

Comment:
Maybe add in the futur a SprintDetail view as an intermediate like we already discussed

Reply:
Why an intermediate ? Maybe just rebame this as SprintDetail no ?

Reply:
To have a great separation of concern and perfectly match with the Master-Detail concept.
If there are a SprintDetail view and a StoryListView we'd have  
```swift
SprintDetail {
  NavigationView {
     VStack { 
          Header()
          StoryList()
    }
    Text("Select a Story")
  }
}
```

Reply:
I think it increases readability and simplify the StoryList view which is just a List basically